### ISSUE TITLE
recipy command fails on Windows due to blessings use

### ISSUE THREAD

**Issue by robintw** [recipy/recipy#141]
I thought blessings was compatible with Windows, but it seems not... Lots of errors about `_curses`.

Seems like the best way is to go for `colorama` instead.

**Comment by robintw** [comment]
Useful example of colorama use from my machine at work:

```
>>> import colorama
>>> print(colorama.Style.BRIGHT + 'Hello')
Hello
>>> print(colorama.Style.BRIGHT + 'Hello')
Hello
>>> print(colorama.Style.BRIGHT + 'Hello')
Hello
>>> print(colorama.Style.RESET_ALL + 'Hello')
Hello
>>> print(colorama.Style.BRIGHT + 'Hello')
Hello
```

**Comment by robintw** [comment]
Fixed in 2989db10491579486379584b9839d531f6d9a404
